<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Radiolarite</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="plotSelectorLabel">
          <property name="text">
           <string>Plot:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="plotSelectorBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="startFrequencyEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>0 GHz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Stop:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="stopFrequencyEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="placeholderText">
           <string>10 GHz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpenTouchstoneFile"/>
    <addaction name="actionExportFigure"/>
    <addaction name="actionExportCSV"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <widget class="QMenu" name="menuShow_Grid">
     <property name="tearOffEnabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Show Grid</string>
     </property>
     <addaction name="actionGridMajor"/>
     <addaction name="actionGridMinor"/>
    </widget>
    <addaction name="menuShow_Grid"/>
    <addaction name="actionCopy_to_clipboard"/>
    <addaction name="actionLegend"/>
   </widget>
   <widget class="QMenu" name="menuMath">
    <property name="title">
     <string>Math</string>
    </property>
    <addaction name="actionTime_Domain_Gating"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuMath"/>
  </widget>
  <action name="actionOpenTouchstoneFile">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExportFigure">
   <property name="text">
    <string>Export Figure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc, Esc, Esc</string>
   </property>
  </action>
  <action name="actionAddSubplot">
   <property name="text">
    <string>Add Subplot</string>
   </property>
  </action>
  <action name="actionAddMarker">
   <property name="text">
    <string>Add Marker</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGridMajor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Major</string>
   </property>
  </action>
  <action name="actionGridMinor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Minor</string>
   </property>
  </action>
  <action name="actionCopy_to_clipboard">
   <property name="text">
    <string>Copy to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionZeroReflection">
   <property name="text">
    <string>Set Reflection to 0</string>
   </property>
  </action>
  <action name="actionTime_Domain_Gating">
   <property name="text">
    <string>Time Domain Gating</string>
   </property>
  </action>
  <action name="actionLegend">
   <property name="text">
    <string>Legend</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="text">
    <string>Export CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
